wolfssl/openssl/evp.h

```diff
@@ -240,6 +240,7 @@ enum {
     EVP_PKEY_RSA      = 16,
     EVP_PKEY_DSA      = 17,
     EVP_PKEY_EC       = 18,
+    EVP_PKEY_ED25519  = NID_ED25519,
```

Comment:
It cannot find `NID_ED25519`. Please include `wolfssl/openssl/ec.h`

Reply:
Thanks, David. I will add it. Strangely, on my MacOS setup it works, but on my Linux test setup it does not without the include

Reply:
Turns out asn.h has to be included in evp.h to resolve the NID_ED25519 value.

I pushed the change. Are we okay with this inclusion? @dgarske 

wolfssl/openssl/ec.h:    NID_ED25519 = ED25519k,
wolfssl/wolfcrypt/asn.h:    ED25519k = 256,